frontend/src/app/list/components/shopping-list-dialog/shopping-list-dialog.component.ts

```
@@ -79,6 +80,7 @@ export class ShoppingListDialogComponent {
         return;
       }
 
+      const { name } = this.listForm.getRawValue();
```

Biome (2.4.6)

[error] Shouldn't redeclare 'name'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)